packages/CodingStandard/src/Fixer/DependencyInjection/InjectToConstructorInjectionFixer.php

```diff
@@ -0,0 +1,239 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\DependencyInjection;
+
+use Nette\PhpGenerator\Method;
+use Nette\Utils\Strings;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class InjectToConstructorInjectionFixer implements DefinedFixerInterface
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Constructor injection should be used instead of @inject annotations.',
+            [
+                new CodeSample(
+'<?php
+class SomeClass
+{
+    /**
+     * @inject
+     * @var stdClass
+     */
+    public $property;
+}'),
+            ]
+        );
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return $tokens->isAllTokenKindsFound([T_CLASS, T_DOC_COMMENT, T_PUBLIC, T_VARIABLE]);
+    }
+
+    public function fix(SplFileInfo $file, Tokens $tokens): void
+    {
+        // 1. find annotation @inject
+        foreach ($tokens as $index => $token) {
+            if (! $token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            $doc = new DocBlock($token->getContent());
+            $injectAnnotations = $doc->getAnnotationsOfType('inject');
+            if (! count($injectAnnotations)) {
+                continue;
+            }
+
+            // 1. remove it
+            foreach ($injectAnnotations as $injectAnnotation) {
+                $injectAnnotation->remove();
+            }
+
+            $token->setContent($doc->getContent());
+
+            // 2. make public property private
```

Comment:
Thanks for your note. 

I think I didn't find a nice way how to find if there is a a property.

I'll add the check you wrote, thanks!